src/univariate/continuous/pgeneralizedgaussian.jl

```diff
@@ -124,7 +124,7 @@ function rand(rng::AbstractRNG, d::PGeneralizedGaussian)
     g = Gamma(inv(d.p), 1)
 
     # bernoulli random variable "b" with parameter (1/2).
-    b = Float64(rand(Bernoulli()))
+    b = (-1) ^ (Float64(rand(Bernoulli())))
```

Comment:
I think this would be clearer by multiplying the range
```suggestion
    b = 2.0 * rand(Bernoulli()) -1
```